components/script/script_module.rs

```diff
@@ -706,23 +708,23 @@ impl ModuleTree {
                     self.set_status(ModuleStatus::Finished);
                     debug!(
                         "Module {} doesn't have module record but tried to load descendants.",
-                        self.url.clone()
+                        &self.url
                     );
                     return;
                 },
                 // Step 5.
                 Some(raw_record) => self.resolve_requested_module_specifiers(
                     &global,
                     raw_record.handle(),
-                    self.url.clone(),
+                    &self.url,
                 ),
             }
         };
 
         match specifier_urls {
             // Step 3.
             Ok(valid_specifier_urls) if valid_specifier_urls.is_empty() => {
-                debug!("Module {} doesn't have any dependencies.", self.url.clone());
+                debug!("Module {} doesn't have any dependencies.", &self.url);
```

Comment:
I suspect you don't need to use a reference here and can just pass `self.url` directly to `debug!()`. It likely does a borrow.

Reply:
Yes, using a reference here is redundant; thanks for catching it 🙏 I've updated it in the latest commit.